src/gui/search/searchwidget.cpp

```diff
@@ -161,6 +162,12 @@ bool SearchWidget::eventFilter(QObject *object, QEvent *event)
         {
             closeTab(tabIndex);
             return true;
+        } else if (mouseEvent->button() == Qt::RightButton)
+        {
+            QMenu m;
+            m.addAction(tr("Close tab"), [=](){ closeTab(tabIndex); return true; });
+            m.addAction(tr("Close all tabs"), [this](){ closeAllTabs(); });
+            m.exec(QCursor::pos());
```

Comment:
use `menu.popup(QCursor::pos());`
refer to `void TagFilterWidget::showMenu(QPoint)` as an example.

Reply:
thanks a lot, very informative! I did everything as i can see